### ISSUE TITLE
cannot compare string

### ISSUE THREAD

**Issue by mozbugbox** [atsepkov/RapydScript#211]
cannot compare `"a" <= "b"` now?
```
Welcome to the RapydScript v0.5.60 REPL! Press Ctrl+C twice or use quit() to quit.
You can use the `repl` object to check/modify settings specific to this prompt.
Use repl.showJavascript=False to stop the REPL from showing the compiled JavaScript.

>>> 'a' < 'b'
1:9: cannot perform binary '<' operation on incompatible elements of type String and String
>>> 
```

**Comment by valq7711** [comment]
Try [this branch](https://github.com/atsepkov/RapydScript/tree/valq7711-fix%23211), don't forget to recompile Rapydscript self - `bin/rapydscript --self` for the changes to take effect

**Comment by mozbugbox** [comment]
thanks, your fix works!

**Comment by valq7711** [comment]
fixed #212
